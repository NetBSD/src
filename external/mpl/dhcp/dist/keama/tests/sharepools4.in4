# DHCPv4 two pools and subnets shared-network declaration config

# shared-network declaration
shared-network "foobar" {
    # interface
    interface "en0";
    # option
    option domain-search "example.com", "example.org";
    # parameter
    default-lease-time 1800;
    # subnet declaration
    subnet 10.5.5.0 netmask 255.255.255.224 {
        # redefined parameter
        default-lease-time 3600;
        # another option
        option ip-forwarding true;
    }
    # second subnet declaration
    subnet 10.10.10.0 netmask 255.255.255.224 { }
    # pools at shared-network level
    pool {
        range 10.5.5.5 10.5.5.10;
    }
    pool {
        range 10.10.10.5 10.10.10.10;
    }    
}
