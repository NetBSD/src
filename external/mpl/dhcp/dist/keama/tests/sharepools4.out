{
  # DHCPv4 two pools and subnets shared-network declaration config
  # shared-network declaration
  "Dhcp4": {
    "interfaces-config": {
      "interfaces": [
        "en0"
      ]
    },
    /// Kea shared-networks are different, cf Kea #236
    "shared-networks": [
      {
        "name": "foobar",
        "subnet4": [
          # subnet declaration
          {
            "id": 1,
            "subnet": "10.5.5.0/27",
            "valid-lifetime": 3600,
            "option-data": [
              # another option
              {
                "space": "dhcp4",
                "name": "ip-forwarding",
                "code": 19,
                "data": "true"
              }
            ],
            "pools": [
              # pools at shared-network level
              {
                "pool": "10.5.5.5 - 10.5.5.10"
              }
            ]
          },
          # second subnet declaration
          {
            "id": 2,
            "subnet": "10.10.10.0/27",
            "pools": [
              {
                "pool": "10.10.10.5 - 10.10.10.10"
              }
            ]
          }
        ],
        "interface": "en0",
        "option-data": [
          # interface
          # option
          {
            "space": "dhcp4",
            "name": "domain-search",
            "code": 119,
//          "original-data": "\"example.com\", \"example.org\"",
            "data": "example.com, example.org"
          }
        ],
        "valid-lifetime": 1800
      }
    ]
  }
}
